source/Junkership.js

```diff
@@ -59,7 +60,17 @@ export default class Junkership extends Pixi.Sprite {
         }
 
         if(Keyb.isJustDown("<space>")) {
-            this.shoot()
+            this.fire(this.position.x,this.position.y,this,
```

Comment:
Not sure why all of this is getting passed. If fire() is a method of Junkership, why is it getting passed a Junkership and several of its attributes? Shouldn't we be able to just call this.fire()?